.babelrc

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": [
+    [
+      "env", {
+        "targets": {
+          "electron": "1.6.6"
```

Comment:
Does this need changing, as #1784 has been merged?

Reply:
Yup! I noticed yesterday and changed it while working on the file selector (based on this PR). I'll push it.